pkgs/tools/filesystems/afpfs-fuse/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, libgcrypt, gmp, readline, fuse }:
+
+stdenv.mkDerivation rec {
+  name = "afpfs-ng-fuse-0.8.2";
+
+  src = fetchgit {
+    url = "https://github.com/chetant/afpfs-ng.git";
```

Comment:
Why are you using your own fork instead of the original? The original has a few additional commits after this revision as well.